dumb-jump.el

```diff
@@ -221,13 +221,19 @@ or most optimal searcher."
                  "(cl-defun test-blah\n"  "(defun tester (blah)"))
 
     (:type "variable" :supports ("ag" "grep" "rg" "git-grep") :language "elisp"
-           :regex "\\\(defvar\\b\\s*JJJ\\j" :tests ("(defvar test " "(defvar test\n"))
+           :regex "\\\(defvar\\b\\s*JJJ\\j"
+           :tests ("(defvar test " "(defvar test\n")
+           :not ("(defvar tester"))
 
     (:type "variable" :supports ("ag" "grep" "rg" "git-grep") :language "elisp"
-           :regex "\\\(defcustom\\b\\s*JJJ\\j" :tests ("(defcustom test " "(defcustom test\n"))
+           :regex "\\\(defcustom\\b\\s*JJJ\\j"
+           :tests ("(defcustom test " "(defcustom test\n")
+           :not ("(defcustom tester"))
+
 
     (:type "variable" :supports ("ag" "grep" "rg" "git-grep") :language "elisp"
-           :regex "\\\(setq\\b\\s*JJJ\\j" :tests ("(setq test 123)") :not ("setq test-blah 123)"))
+           :regex "\\\(setq\\b\\s*JJJ\\j" :tests ("(setq test 123)")
```

Comment:
Like in the other places, it would be nice with a newline before `:tests` for readability.